etc/flux.service.in

```diff
@@ -7,7 +7,7 @@ TimeoutStopSec=90
 KillMode=mixed
 ExecStart=@X_BINDIR@/flux broker \
   --config-path=@X_SYSCONFDIR@/flux/system/conf.d \
-  --k-ary=256 \
+  -Stbon.fanout=256 \
```

Comment:
should the commit message header be "rename tbon.arity"?   Preferable is "rename tbon.arity to tbon.fanout", but I assume that would make the line too long.

Reply:
Heh, in the past I've felt the new name would have more significance than the old one in the history.  In this case, I think maybe a generic message that doesn't mention names is probably better.  Will psuh an update.